.github/workflows/deploy.yml

```diff
@@ -1,38 +1,300 @@
+        with:
+          openshift_server_url: ${{env.OPENSHIFT_SERVER_GOLDDR}}
+          openshift_token: ${{env.OPENSHIFT_GOLDDR_TOKEN}}
+          insecure_skip_tls_verify: true
+          namespace: ${{env.DEFAULT_NAMESPACE}}
 
-      # Runs a set of commands using the runners shell
-      - name: Run a multi-line script
+      # Push the image to selected repository
+      - name: Push Image
         run: |
-          echo Add other actions to build,
-          echo test, and deploy your project.
+          oc tag ${{env.IMAGE_NAME}}:${{needs.validate.outputs.IMAGE_TAG}} ${{needs.validate.outputs.TARGET_NAMESPACE}}/${{env.IMAGE_NAME}}:${{needs.validate.outputs.IMAGE_TAG}}
 
+  deploy-feature-build:
+    if: github.event.inputs.Build_Type == 'Feature'
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+    needs: [validate , prepare]
 
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+
+      # 5. Edit the selcted Argo CD manifest file
+      # Update the image ID in the manifest repository used by Argo CD.
+      # We need to use the SSH deploy key to check out the manifest repo, set up
+      # Kustomize in the runner, update the image ID, commit the change, and
+      # push the changes back to the manifest repository.
+      # ------------------------------------------------------------------------
+      - name: Prepare deploy key
+        uses: webfactory/ssh-agent@v0.5.3
+        with:
+          ssh-private-key: ${{secrets.GITOPS_C8D6F2_PRIVATE_KEY}}
+
+      - name: Check out manifest repo
+        uses: actions/checkout@v3
+        with:
+          ssh-key: ${{secrets.GITOPS_C8D6F2_PRIVATE_KEY}}
+          repository: bcgov-c/tenant-gitops-c8d6f2
+          ref: 7537-move-rtrans-to-gold-cluster-for-dev-and-test
+
+      - name: Set up Kustomize
+        uses: imranismail/setup-kustomize@v1
+
+      - name: Determine Kustomize folder location
+        id: get-kustomize-folder
+        uses: actions/github-script@v5
+        with:
+          script: |
+            if (`${{github.event.inputs.ENVIRONMENT}}` == 'dev') {
+              return "dev"
+            }
+            if (`${{github.event.inputs.ENVIRONMENT}}` == 'test') {
+              return "test"
+            }
+            core.setFailed(`❌ Invalid inputs for Environment and branch`  );
+
+      - name: Update image ID and commit change
+        if: ${{success()}}
+        run: |
+          cd rtrans-devops/overlay/${{steps.get-kustomize-folder.outputs.result}}
+          kustomize edit set image "rtrans-image=${{env.IMAGE_REGISTRY_GOLD}}/${{needs.validate.outputs.TARGET_NAMESPACE}}/${{env.IMAGE_NAME}}@${{needs.validate.outputs.IMAGE_ID}}"
+          cd ../${{steps.get-kustomize-folder.outputs.result}}-dr
+          kustomize edit set image "rtrans-image=${{env.IMAGE_REGISTRY_GOLDDR}}/${{needs.validate.outputs.TARGET_NAMESPACE}}/${{env.IMAGE_NAME}}@${{needs.validate.outputs.IMAGE_ID}}"
```

Comment:
added logic to check replication in hnsesb pipeline.. need to add the same here

Reply:
Added logic via steps, one conditional, to handle only deploying to golddr for prod